src/cbmc/CMakeLists.txt

```diff
@@ -16,6 +16,7 @@ target_link_libraries(cbmc-lib
     goto-instrument-lib
     goto-programs
     goto-symex
+    jsil
```

Comment:
See thread above - requested by @tautschnig due to inconsistent use.

Reply:
I agree that it would better fit a separate PR, but that PR would necessarily depend on this one because of overlapping changes, hence adding it in here seemed ok to me.